build/test/TestPackageProjects.props

```diff
@@ -0,0 +1,160 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project DefaultTargets="Layout" ToolsVersion="14.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
```

Comment:
Still an issue

---
In reply to: [69724165](https://github.com/dotnet/cli/pull/3779#discussion_r69724165) [](ancestors = 69724165)